fcrepo-webapp/src/main/resources/spring/master.xml

```diff
@@ -3,14 +3,13 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="
     http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
-    
+
     <!-- Master context for fcrepo4. -->
 
     <import resource="${fcrepo.spring.repo.configuration:classpath:/spring/repo.xml}"/>
     <import resource="${fcrepo.spring.rest.configuration:classpath:/spring/rest.xml}"/>
-    <import resource="${fcrepo.spring.minter.configuration:classpath:/spring/minter.xml}"/>
```

Comment:
It sounds like https://github.com/fcrepo4-exts/fcrepo-webapp-plus/pull/30 should be updated based on this discussion, no?